lib/src/rules/avoid_renaming_method_parameters.dart

```diff
@@ -59,16 +59,21 @@ class AvoidRenamingMethodParameters extends LintRule implements NodeLintRule {
       return;
     }
 
-    var visitor = _Visitor(this, context);
+    var visitor = _Visitor(this);
     registry.addMethodDeclaration(this, visitor);
   }
 }
 
 class _Visitor extends SimpleAstVisitor<void> {
+  static const LintCode parameterCode = LintCode(
+      "avoid_renaming_method_parameters",
+      "The overridden method should have same parameter names as the parent method.",
+      correction:
+          "The parameter '{0}' should have the name '{1}' to match the name used in the overridden method.");
```

Comment:
The message you currently have as the correction message would be good as the problem message, but the correction message should take the form "Try <solution>.", so perhaps "Try renaming the parameter.".